.github/workflows/lint.yml

```diff
@@ -82,3 +81,9 @@ jobs:
         with:
           command: fmt
           args: --all -- --check
+
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: reviewdog/action-actionlint@v1.21.0
```

Comment:
Possible bug: this action only runs when the lint workflow file changes, but we want it to run when any workflow file changes.

Or possibly any file in `.github` ?

Reply:
I think only files in `.github/workflows/`

Reply:
Are the mergify and dependabot changes in this PR from the linter?
Or are they manual changes?

Reply:
@teor2345 yes, all the changes are from the linter. Except the `macos-latest` change, as that was required to be manually changed as this is the actual name used by GHA and required to be changed by the linter

Reply:
Fixed here: https://github.com/ZcashFoundation/zebra/pull/3940/commits/22543b71c8e42763d0ffd24022b3f0c6afa9ff11